code/index.html

```diff
@@ -30,8 +36,13 @@
 
     <h1 id="mystery"></h1>
 
-    <button>REVEAL THE CRIME</button>
+    <button onclick="revealMystery()" id="clickMe">Reveal the crime</button>
```

Comment:
I think Maks or Van (really unsure who of them) once said that you should put the id-attribute first, then class, then the other attribute-stuff.